### PR TITLE
Fix doc typos

### PR DESCRIPTION
**Pull request by jswanner**
Thanks again for the amazing library!

Hopefully the changes to the code examples are what you intended.

### CODE REVIEW

File: lib/thousand_island/handler.ex [hunk 1]
```diff
@@ -71,7 +71,7 @@ defmodule ThousandIsland.Handler do
 
     @impl ThousandIsland.Handler
     def handle_data(msg, _socket, state) do
-      IO.puts(msg)
+      send(self(), {:send, msg})
```
Comment (mtrudel) [inline lib/thousand_island/handler.ex:74]: Not quite, but I can see the confusion! The `handle_info({:send, msg}, {socket, state})` call below is meant to be called from somewhere else in your application (or even from an `iex` session) - it sends data *to* the connected client. The `handle_data(msg, _socket, state)` function is called by Thousand Island when data is received *from* the connected client. In an actual implementation, this log message would be replaced by something 'real'; the logging here is correct in as far as this is a super simple demo.

File: lib/thousand_island/handler.ex [hunk 2]
```diff
@@ -103,7 +103,7 @@ defmodule ThousandIsland.Handler do
 
     @impl ThousandIsland.Handler
     def handle_connection(socket, state) do
-      %{address: host} = ThousandIsland.Socket.peer_info(socket)
+      %{address: address} = ThousandIsland.Socket.peer_info(socket)
```
Comment (mtrudel) [inline lib/thousand_island/handler.ex:106]: This is correct; good catch!

**Comment by mtrudel** [comment]
I backed out the one incorrect change (and commented on it above to clear up the confusion)Thanks for the PR!

File: lib/thousand_island/handler.ex (hunk 1 above)
Comment (jswanner) [reply lib/thousand_island/handler.ex]: Sorry to give you more work to do in reverting this change.

**Pull request merged**